### ISSUE TITLE
[bug spike] SC | Investigate ITF failures

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#61377]
## Value Statement
**_As a_** Veteran filing for a Supplemental Claim
**_I want to_** not get blocked by Intent-to-file (ITF) creation failures
**_So that_** I can proceed with filling out the form

---
## Background Context

We're monitoring our Supplemental Claim form (20-0995) Sentry events and noticing excessive [ITF creations failures](http://sentry.vfs.va.gov/organizations/vsp/discover/results/?field=message&name=Supplemental+Claims&project=4&query=level%3Aerror+message%3A%22itf_creation_failed+https%3A%2F%2Fwww.va.gov%2Fdecision-reviews%2Fsupplemental-claim%2Ffile-supplemental-claim-form-20-0995%2F%2A%22&sort=-message&statsPeriod=24h&widths=-1) (345 in the past 2 weeks; 1.26k in 90 days). We need to make sure these failures are transient and not blocking the Veteran from starting the Supplemental Claim process.

This ITF endpoint is shared with form 526 and will soon be switched over to use Lighthouse's new endpoint (switch within `vets-api`), so we need to ensure these ITF are not blocking the Veteran and ensure we have monitoring (Datadog) in place for after the switch.

## Acceptance Criteria
- [x] ITF failures investigated
- [ ] Continued ITF failures should not block starting the form
- [ ] Veterans are not blocked from filing a Supplemental Claim 
- [x] Monitoring is in place and adequate for catching blocking errors

## Out of scope
- Add here

## Open questions
Discuss the UX:
  - Should we block submission instead of blocking starting the form if an ITF is not filed?
  - Should a creation error be shown and inform the Veteran to file an ITF is one hasn't already been filed?
  - Should we add that an ITF _was_ created for them on the confirmation page?

## Tasks
- [ ] Investigate ITF creation errors - are they intermittent?
- [ ] Ensure appropriate logging & monitoring is in place in Datadog
- [ ] Discuss UX

## Definition of Ready 

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Reviewed and approved by product and/or design

**Comment by saderagsdale** [comment]
@kylesoskin @Mottie I'm going to check with the business on whether or not they want a submission blocked if an ITF fails to generate.

**Comment by Mottie** [comment]
From looking at Sentry logs, 526 has _way more_ ITF creation failures than Supplemental Claims. Also, the pensions app has a note on the intro page and confirmation page (I think) about submitting an ITF - ITF submission is not built into that app at all.

**Comment by Mottie** [comment]
I discussed a possible solution with @rocio-desantiago yesterday - basically, for every successful ITF API call, we save the date checked and the ITF expiration date. If at any time the ITF API call fails and the checked date is recent (maybe within the past week) and the expiration date hasn't gone by, we show an alert stating that the ITF check failed, but allow them to proceed. Then upon submission, we can add an alert that they will need to submit an ITF if the saved data doesn't include an active ITF. @eileen-coforma What do you think? If you think this solution works, we can discuss updating the design & content.

**Comment by eileen-coforma** [comment]
@saderagsdale is this blocked until we get a response from Karla?

**Comment by saderagsdale** [comment]
Note from Karla Leal (VBA):

_

> Hi Sade, there is a policy restriction in allowing the user to complete the supplemental claim in VA.gov without the ITF.
>  
> If we allow the user to start he supplemental claim form and they do not finish the form and without an ITF, it can impact their benefits effective date.
>  
> Let me know if you have any other questions.

_

**Comment by saderagsdale** [comment]
- One option is to wait for the migration to be completed. Need to find out when the 526 team will complete this. 
- Another is to reach out to EVSS. That's probably the slower one.
